.github/ISSUE_TEMPLATE/bug-or-issue.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright The Notary Project Authors.
```

Comment:
Please change the first line to match other Community files within Veraison

Reply:
I am trying to find an example for the text to use. Will the following work:

> Copyright The Mozilla Maintainers and Contributors and The Veraison Go-COSE Authors

or just 

> Copyright The Veraison Go-COSE Authors

Reply:
Thanks @toddysm 

> Copyright The Veraison Go-COSE Authors

As this project as now decoupled from the Mozilla maintainers.

Reply:
Fixed the copyright notice.